.github/workflows/archive_closed_jobs.yml

```diff
@@ -4,6 +4,7 @@ on:
   schedule:
     # 1000 UTC will always be after midnight ET.
     - cron: 0 10 * * *
+  pull_request:
```

Comment:
[suggestion] I think we probably don't want to include `pull_request` on this job, to prevent unexpected side effects to other changes.

On the good side, it did prove that the changes worked: https://github.com/18F/join.tts.gsa.gov/tree/archiver/auto/archive

Reply:
Yeah, that was the plan! mostly just to test.